app/services/bulk_upload/lettings/year2023/row_parser.rb

```diff
@@ -329,9 +329,26 @@ class BulkUpload::Lettings::Year2023::RowParser
             },
             on: :after_log
 
+  validates :field_15,
+            presence: {
+              if: proc { [2, 4, 6, 8, 10, 12].include?(field_5) && log_uses_old_scheme_id? },
```

Comment:
would we be able to check bulk_upload.needstype here instead? Might be a bit clearer what these numbers mean

Reply:
I like the name `log_uses_old_scheme_id`

Reply:
Yep, sure!